linera-execution/src/runtime.rs

```diff
@@ -1611,8 +1611,8 @@ impl ServiceSyncRuntime {
         context: QueryContext,
         deadline: Option<Instant>,
     ) -> Self {
-        // Query the allow_contract_logs setting from the execution state.
-        let allow_contract_logs = execution_state_sender
+        // Query the allow_application_logs setting from the execution state.
+        let allow_application_logs = execution_state_sender
             .send_request(|callback| ExecutionRequest::AllowContractLogs { callback })
```

Comment:
The request should probably also be `AllowApplicationLogs` now.

Reply:
Ah, done in [One more rename](https://github.com/linera-io/linera-protocol/pull/5099/commits/a2da76d312a3f9c092f2fde86412a593a7103524).